lib/libc/minimal/include/time.h

```diff
@@ -34,10 +34,7 @@ struct tm {
 typedef int64_t time_t;
 typedef int32_t suseconds_t;
 
-struct timespec {
-	time_t tv_sec;
-	long tv_nsec;
-};
+#include <sys/_timespec.h>
```

Comment:
How does this work, where is <sys/_timespec.h> coming from?

Reply:
(maybe the commit order is being shown oddly by github)

Reply:
@galak https://github.com/isaacs/github/issues/386#issuecomment-494288396